CMakeLists.txt

```
@@ -17,7 +17,12 @@ include_directories(${SDL2_INCLUDE_DIRS})
 add_subdirectory(lvgl)
 add_executable(main main.c mouse_cursor_icon.c ${SOURCES} ${INCLUDES})
 add_compile_definitions(LV_CONF_INCLUDE_SIMPLE)
-target_link_libraries(main PRIVATE lvgl lvgl::examples lvgl::demos ${SDL2_LIBRARIES})
+target_link_libraries(main PRIVATE lvgl lvgl::examples lvgl::demos ${SDL2_LIBRARIES} m)
+if(NOT LV_CONF_BUILD_DISABLE_THORVG_INTERNAL)
```

```suggestion
if(LV_CONF_BUILD_THORVG_EXTERNAL)
```
to add ThorVG only if explicitly requested.